.rubocop.yml

```diff
@@ -0,0 +1,188 @@
+  Description: "Use Unix-style line endings."
+  StyleGuide: "#crlf"
+  Enabled: true
+
+Layout/ExtraSpacing:
+  Description: "Do not use unnecessary spacing."
+  Enabled: true
+
+Layout/LeadingCommentSpace:
+  Description: "Comments should start with a space."
+  StyleGuide: "#hash-space"
+  Enabled: true
+
+Layout/SpaceAfterColon:
+  Description: "Use spaces after colons."
+  StyleGuide: "#spaces-operators"
+  Enabled: true
+
+Layout/SpaceAfterComma:
+  Description: "Use spaces after commas."
+  StyleGuide: '#spaces-operators'
+  Enabled: true
+
+Layout/SpaceAfterMethodName:
+  Description: >-
+                 Do not put a space between a method name and the opening
+                 parenthesis in a method definition.
+  StyleGuide: "#parens-no-spaces"
+  Enabled: true
+
+Layout/SpaceAfterNot:
+  Description: Tracks redundant space after the ! operator.
+  StyleGuide: "#no-space-bang"
+  Enabled: true
+
+Layout/SpaceAfterSemicolon:
+  Description: "Use spaces after semicolons."
+  StyleGuide: '#spaces-operators'
+  Enabled: true
+
+Layout/SpaceBeforeComma:
+  Description: "No spaces before commas."
+  Enabled: true
+
+Layout/SpaceBeforeComment:
+  Description: >-
+                 Checks for missing space between code and a comment on the
+                 same line.
+  Enabled: true
+
+Layout/SpaceBeforeSemicolon:
+  Description: "No spaces before semicolons."
+  Enabled: true
+
+Layout/SpaceAroundOperators:
+  Description: "Use a single space around operators."
+  StyleGuide: "#spaces-operators"
+  Enabled: true
+
+Layout/Tab:
+  Description: "No hard tabs."
+  StyleGuide: "#spaces-indentation"
+  Enabled: true
+
+Layout/TrailingBlankLines:
+  Description: "Checks trailing blank lines and final newline."
+  StyleGuide: '#newline-eof'
+  Enabled: true
+
+Layout/TrailingWhitespace:
+  Description: "Avoid trailing whitespace."
+  StyleGuide: "#no-trailing-whitespace"
+  Enabled: true
+
+#################### Naming ##############################
+
+Naming/ClassAndModuleCamelCase:
+  Description: "Use CamelCase for classes and modules."
+  StyleGuide: "#camelcase-classes"
+  Enabled: true
+
+Naming/ConstantName:
+  Description: "Constants should use SCREAMING_SNAKE_CASE."
+  StyleGuide: "#screaming-snake-case"
+  Enabled: true
+
+#################### Style ###############################
+
+Style/ClassVars:
+  Description: 'Avoid the use of class variables.'
+  StyleGuide: '#no-class-vars'
+  Enabled: true
+
+Style/DefWithParentheses:
+  Description: "Use def with parentheses when there are arguments."
+  StyleGuide: "#method-parens"
+  Enabled: true
+
+Style/Encoding:
+  Description: "Use UTF-8 as the source file encoding."
+  StyleGuide: "#utf-8"
+  Enabled: true
+
+# TODO
+Style/FrozenStringLiteralComment:
+  Description: "Add frozen_string_literal comment to the top of files"
+  Include:
+  - "app/**/*.rb"
+  - "lib/**/*.rb"
+  Enabled: false
+
+Style/HashSyntax:
+  Description: "Prefer Ruby 1.9 hash syntax { a: 1, b: 2 } over 1.8 syntax"
```

Comment:
Maybe add a beginner friendly ` { :a => 1, :b => 2 }` after `1.8 syntax` in the description?